frontend/src/app/components/master-page/master-page.component.html

```diff
@@ -62,6 +62,21 @@ <h6 *ngIf="env.LIQUID_ENABLED || env.BISQ_ENABLED" class="dropdown-header" i18n=
 </nav>
 </header>
 
+<div *ngIf="network.val === 'testnet' || network.val === 'signet'">
+  <div class="container p-lg-0 pb-0" style="max-width: 100%; margin-top: 7px" *ngIf="storageService.getValue('hideWarning') !== 'hidden'">
+    <div class="alert alert-danger mb-0 text-center">
+      <div class="d-flex justify-content-center align-items-center">
+        <fa-icon class="between-arrow mr-1" [icon]="['fas', 'exclamation-triangle']" [fixedWidth]="true"></fa-icon>
+        Bitcoin {{ network.val | titlecase }} is used for testing - Coins have no value
```

Comment:
We should make this an i18n key so it can be translated. But we might need to find a way that makes sense to add the network as a variable. Or change to a generic title that says "this is a Testnet" even if it's Signet or Liquid testnet etc.

Reply:
<img width="355" alt="image" src="https://user-images.githubusercontent.com/9780671/226093020-266d09e1-558e-4c14-ae6b-c1069eec5fb8.png">